unstructured_inference/models/tables.py

```diff
@@ -608,9 +608,25 @@ def structure_to_cells(table_structure, tokens):
     return cells, confidence_score
 
 
+def fill_cells(cells: List[dict]) -> List[dict]:
+    """add empty cells to pad cells that spans multiple rows for html conversion
```

Comment:
can you describe the input data structure here which will make this code a bit easier to follow?